src/events/stickyMessage.ts

```diff
@@ -0,0 +1,65 @@
+import { Events } from 'discord.js'
+
+import { StickyMessage } from '@prisma/client'
+
+import {
+  isChannelLock,
+  lockChannel,
+  unlockChannel,
+} from '../features/stickyMessage/lockChannel.js'
+import {
+  getCounter,
+  incCounter,
+  isNeedToUpdateMessage,
+  resetCounter,
+} from '../features/stickyMessage/messageCooldown.js'
+import { prisma } from '../prisma.js'
+import { defineEventHandler } from '../types/defineEventHandler.js'
+import { getCache, saveCache } from '../utils/cache.js'
+
+export default defineEventHandler({
+  eventName: Events.MessageCreate,
+  once: false,
+  execute: async (_botContext, message) => {
+    const stickyMessage = getCache(message.channelId) as StickyMessage
```

Comment:
`getCache()`s and `saveCache()`s shouldn't use just `message.channelId` for cache keys, as the cache is not specific to sticky message feature. Consider adding `sticky-` prefix or something to cache key so other features can utilize the cache too.

Reply:
u forgot to update `getCache()`s too lol